samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -767,7 +767,8 @@ Using aggressive configurations can reduce air time availability for all streams
 SD card setup
 *************
 
-The sample only supports pre-encoded LC3 data stored as LC3 files on an SD card.
+The sample supports pre-encoded LC3 data stored as LC3 files on an SD card.
```

Comment:
```suggestion
This sample can support pre-encoded LC3 data stored as LC3 files on an SD card.
```
If the SD card is not mandatory anymore, update also lines 15-18 of this document so that the SD card is not listed as a limitation but as an option.